src/node_jsvmapi.cc

```diff
@@ -217,15 +217,15 @@ namespace v8impl {
       }
 
       virtual void* Data() override {
-        return _cbdata->GetInternalField(kDataIndex).As<v8::External>()->Value();
+        return _cbdata->GetInternalField(kDataIndex).template As<v8::External>()->Value();
```

Comment:
Should this use Cast like v8::Local<v8::External>::Cast() instead of the .template ?